### DISCUSSION TITLE
Catch-22 aws-apigatewayv2-alpha HttpApi < > HttpAuthorizer

### DISCUSSION THREAD

**Discussion by gabriels1234** [aws/aws-cdk#25681] [Q&A]
Simple question (aws-apigatewayv2-alpha):
When creating an HttpAPI it is possible to specify a default_authorizer.

Meanwhile, to create an HttpAuthorizer you need to specify the http_api.

Any ideas?

**Comment by indrora** [comment]
Could you please specify which HttpAuthorizer you're talking about? The HttpAuthorizer inside that package is only a stub -- you'll want one of the authorizers in the -authorizers package: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-apigatewayv2-authorizers-alpha-readme.html
